packages/cli/src/oclif/ZapierBaseCommand.js

```diff
@@ -90,7 +90,11 @@ class ZapierBaseCommand extends Command {
 
   // validate that user input looks like a semver version
   throwForInvalidVersion(version) {
-    if (!version.match(/^\d+\.\d+\.\d+$/g)) {
+    if (
+      !version.match(
+        /^(0|[1-9]\d{,2})\.(0|[1-9]\d{,2})\.(0|[1-9]\d{,2})(-[0-9A-Za-z]+)*$/g,
```

Comment:
Based on https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string, but still limited to `999.999.999` and not allowing `.` or `+` in labels.